Assets/Developers/Lab/Systems/SpawnSystem.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Unity.Burst;
+using Unity.Collections;
+using Unity.Entities;
+using Unity.Jobs;
+using Unity.Mathematics;
+using Unity.Transforms;
+
+namespace JanusVagabond
+{
+
+  [UpdateInGroup(typeof(SimulationSystemGroup))]
+  public class SpawnSystem : JobComponentSystem
+  {
+    const int RANDOM_SEED = 999;
+    const float GROUND_HEIGHT = -2f;
+    static Random m_RandomPool = new Random(RANDOM_SEED);
+
+    BeginInitializationEntityCommandBufferSystem m_EntityCommandBufferSystem;
+    protected override void OnCreate()
+    {
+      // Cache the BeginInitializationEntityCommandBufferSystem in a field, so we don't have to create it every frame
+      m_EntityCommandBufferSystem = World.GetOrCreateSystem<BeginInitializationEntityCommandBufferSystem>();
+    }
+
+    // [BurstCompile]
```

Comment:
Not sure why you commented out all the `BurstCompile`s. Is there a reason for it? I would probably remove them if the feature doesn't work

Reply:
Talked a bit about this during last call. The BurstCompiler doesn't work with Job that deals with Transform for some reason (only LocalToWorld). Just leaving it there to eventually enable it once they support more type. Will add a comment on it.